### ISSUE TITLE
No Learning rate decay in the training code

### ISSUE THREAD

**Issue by z1277724487** [Baymax-chen/EMVD#8]
I didn't find any learing rate adjustment in the training code.
